public-ui/src/styles.css

```diff
@@ -611,30 +623,31 @@ a a:link a:visited {
   word-wrap: break-word;
   white-space: normal;
   background-color: #ffffff;
-  box-shadow: 0 4px 6px 0 rgba(0, 0, 0, 0.15);
   font-size: 14px;
   padding: 5px;
   font-family: GothamBook, Arial, sans-serif;
-  border: 1px solid #262262;
+  padding: .5rem;
```

Comment:
Just noticed there's also a `padding` property for this selector above on line 627